docker/README.md

```diff
@@ -0,0 +1,156 @@
+(SIGINT). Alternatively, run:
+```console
+$ docker-compose down
+```
+
+This will tear down the environment, so you&rsquo;ll have a lengthy wait when
+starting the containers next time. If you&rsquo;re going to need the environment
+again soon, consider instead using the equivalent of standby:
+
+```console
+$ docker-compose pause
+$ docker-compose unpause
+```
+
+To only run the infrastructure containers (like PostgreSQL, AWS and ClamAV),
+or to run the other HOCS services that aren&rsquo;t frontend, do:
+```console
+$ ./scripts/infrastructure.sh
+$ ./scripts/services.sh
+```
+
+You can also run individual services. For example:
+```console
+$ docker-compose up frontend # localhost:8080
+```
+
+If you ask to bring up a service with a dependency, Compose will treat this
+command as an implicit request to bring up the dependencies before bringing
+up the dependant. If you don&rsquo;t want that, use:
+
+```console
+$ docker-compose up --no-deps frontend
+```
+
+If Docker doesn&rsquo;t have the images it needs, it will automatically
+pull them. However, it won&rsquo;t pull existing images. If you want to run
+the latest versions of services, remember to ask for them first:
+
+```console
+$ docker-compose pull
+$ docker-compose up
+```
+
+## Troubleshooting
+### Unhealthy containers
+
+Docker Compose is configured so that containers that depend on other containers
+don't get started until their dependencies are not only running, but also ready
+for use.
+
+If a container doesn't start working after a certain period of time, it will be
+marked as &ldquo;unhealthy&rdquo; and Docker Compose will exit.
+
+The error mesage will look something like this:
+
+```console
+ERROR: for frontend  Container "fe34db64b428" is unhealthy.
+ERROR: Encountered errors while bringing up the project.
+```
+
+The hexadecimal number is the problematic container's ID.
+You can check to see which container is misbehaving with:
+
+```console
+$ docker container ls
+$ docker container logs fe34db64b428
+```
+
+(If you&rsquo;re new to Docker, the duration under &ldquo;STATUS&rdquo; is
+generally more applicable here than the one under &ldquo;CREATED&rdquo;.)
+
+This error is often due the process simply taking longer than usual to start,
+and waiting a bit longer is enough to resolve the issue.
+
+However, if you grow tired of waiting, or the logs look strange, you can usually
+fix the problem by restarting the container:
+
+```console
+$ docker container restart fe34db64b428
+```
+
+Once the container is marked as &ldquo;healthy&rdquo; you can just pick the
+the Docker Compose startup from where you left off:
+
+```console
+$ docker-compose up
+```
+
+For more serious problems you might need to tear down the environment with
+`docker-compose down` and start again.
+
+### Missing SQL tables
+
+The info-service doesn&rsquo;t come with its schema, so if you&rsquo;re setting
+up your environment for the first time your info-service you&rsquo;ll need to
```

Comment:
Think there's a word missing here "your environment for the first time your info-service you'll..."

Reply:
Fixed. Looks like I got my tenses confused.
It's not actually "the first time you run your environment", but more "every time you have a new postgres", so I've reworded it.